cli/src/core/parse/profile.go

```diff
@@ -0,0 +1,46 @@
+package parse
+
+import (
+	"cli/core"
+	"cli/core/state"
+
+	"github.com/luno/jettison/errors"
+	"github.com/spf13/cobra"
+)
+
+func GetPackageSpecFromProfile(cmd *cobra.Command, config core.Config, packageSpec core.PackageSpec) (*core.PackageSpec, error) {
+	profile := core.Profile{}
+	profileName, err := cmd.Flags().GetString("profile")
+	if err != nil {
+		return nil, errors.Wrap(err, "")
+	}
+
+	for _, p := range config.Profiles {
+		if p.Name == profileName {
+			profile = p
+			break
+		}
+	}
+
+	if !cmd.Flags().Changed("dev") && profile.Dev {
+		packageSpec.IsDev = profile.Dev
+	}
+
+	if !cmd.Flags().Changed("only") && profile.Only {
+		packageSpec.IsOnly = profile.Only
+	}
+
+	if len(profile.Packages) > 0 {
+		packageSpec.Packages = append(profile.Packages, packageSpec.Packages...)
+	}
+	if len(profile.EnvFiles) > 0 {
```

Comment:
Spacing to be consistent

Reply:
Done in [154f04b](https://github.com/openhie/package-starter-kit/pull/35/commits/154f04bf730d7df77323c485d2abcc3e41c01182)